package.json

```diff
@@ -2,17 +2,17 @@
   "name": "capi-node-example",
   "type": "module",
   "dependencies": {
-    "capi": "0.1.0-beta.33",
-    "@capi/polkadot": "https://capi.dev/@v0.1.0-beta.33/c51bf258674e5080/polkadot.tar",
-    "@capi/polkadot-dev": "https://capi.dev/@v0.1.0-beta.33/c51bf258674e5080/polkadot-dev.tar"
+    "@capi/polkadot": "https://capi.dev/@v0.1.0-gamma.1/315c963e5803ec98/polkadot.tar",
+    "capi": "0.1.0-gamma.1"
   },
   "devDependencies": {
     "ts-node": "^10.9.1"
   },
   "scripts": {
     "capi": "capi",
-    "sync": "capi sync --package-json package.json",
-    "main": "capi serve -- ts-node-esm index.ts"
+    "sync": "capi sync node",
+    "main": "capi serve -- npm run run",
```

Comment:
This shouldn't require the leading `capi serve --`

Reply:
How about...

```json
// ...
"main": "node --loader ts-node/esm index.ts",
"dev": "capi serve --target dev -- npm run main"
```

Reply:
done in https://github.com/paritytech/capi-node-example/pull/3/commits/3e10c4ea602b1931720b1cd33215a60b7692b495